tools/pipelines/build-docs.yml

```diff
@@ -186,6 +193,34 @@ stages:
             artifact: api-extractor-combined
             path: '$(Build.SourcesDirectory)/_api-extractor-temp/doc-models'
 
+        - ${{ each version in parameters.docVersions.previousVersions }}:
+          - task: Bash@3
+            displayName: Create ${{version}} api-extractor-temp directory
+            inputs:
+              targetType: 'inline'
+              workingDirectory: $(Build.SourcesDirectory)
+              script: |
+                echo "creating api-extractor-temp-${{version}} folder"
+                mkdir -p $(Build.SourcesDirectory)/_api-extractor-temp-${{version}}/doc-models
+
+          - task: AzureCLI@2
+            displayName: 'Download ${{version}} api-extractor JSON blob'
+            continueOnError: true
+            inputs:
+              azureSubscription: 'fluid-docs'
+              scriptType: bash
+              scriptLocation: inlineScript
+              inlineScript: |
+                az storage blob download -f '$(Build.SourcesDirectory)/latest-${{version}}.tar.gz' -c 'api-extractor-json' -n latest-${{version}}.tar.gz --account-name $(STORAGE_ACCOUNT) --account-key $(STORAGE_KEY) --overwrite --verbose
```

Comment:
Can we add some documentation at the root of this file explaining the assumptions we're making here? Devs looking at this pipeline in the future will need to know about the artifact scheme we're depending on here.